modules/icefinder.nf

```diff
@@ -1,19 +1,18 @@
 #!/usr/bin/env nextflow
 nextflow.enable.dsl=2
 
-process ice_finder {
-    publishDir "$launchDir/icefinder_results", mode: 'copy'
+process ICEFINDER {
+    publishDir "$launchDir/$params.outdir/prediction/icefinder_results", mode: 'copy'
     stageInMode = 'copy'
 
-    memory "16 G"
     cpus 1
-
+    memory { 16.GB * task.attempt }
     errorStrategy 'retry'
     maxRetries 3
 
     container "${projectDir}/singularity/icefinder-v1.0-local.sif"
 
-    containerOptions="--bind $PWD/icefinder_results/input.list:/install/ICEfinder_linux/input.list --bind $PWD/icefinder_results/gbk/:/install/ICEfinder_linux/gbk/ --bind $PWD/icefinder_results/tmp/:/install/ICEfinder_linux/tmp/ --bind $PWD/icefinder_results/result/:/install/ICEfinder_linux/result/ --pwd /install/ICEfinder_linux/"
+    containerOptions="--bind $PWD/$params.outdir/prediction/icefinder_results/input.list:/install/ICEfinder_linux/input.list --bind $PWD/$params.outdir/prediction/icefinder_results/gbk/:/install/ICEfinder_linux/gbk/ --bind $PWD/$params.outdir/prediction/icefinder_results/tmp/:/install/ICEfinder_linux/tmp/ --bind $PWD/$params.outdir/prediction/icefinder_results/result/:/install/ICEfinder_linux/result/ --pwd /install/ICEfinder_linux/"
```

Comment:
We need to also consider  docker, `--bind` is for singularity

Reply:
Yes, we are doing this specifically for ICEfinder, which is kind of a problematic tool. In the README file we have the instructions to generate the singularity image of the tool once the user get the license and their own copy of ICEfinder. This is a special case in which we are not using a public container and we use singularity.

Reply:
Right... I remember now. OK, I would keep track of this and in the future we should add the docker version (which may come in handy if we want to run this in the cloud at some point)